appsdk/configurable.go

```diff
@@ -242,11 +242,15 @@ func (dynamic AppFunctionsSDKConfigurable) MQTTSend(parameters map[string]string
 	}
 	dynamic.Sdk.LoggingClient.Debug("MQTT Send Parameters", "Address", addr, Qos, qosVal, Retain, retainVal, AutoReconnect, autoreconnectVal, Cert, cert, Key, key)
 
+	pair := transforms.KeyCertPair{}
```

Comment:
Should only create is cert & key are not empty, otherwise set pair to nil.

Reply:
ok